### PR TITLE
Fix content in `description.md`

### PR DESCRIPTION
**Pull request by razvand**
Fix typos, apply linter checks, correct section links.

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -543,26 +543,26 @@ Exams aim to measure comprehension, critical thinking, and problem-solving skill
...
 
-* **Written Exams** consist of short answers, essays, or problems to solve.
+- **Written Exams** consist of short answers, essays, or problems to solve.
   They are versatile and are fit various subjects.
   Written exams build on what you've learned earlier.
   Written exams build upon previous learning, but they aren't adaptive and don't personalize the experience for each learner.
   Additionally, they might not be the best choice for assessing complex situations, often leaning towards straightforward questions.
 
-* **Multiple-Choice Exams** involve choosing the correct answer from a set of options, essentially being a compilation of [quizzes](#quizzes).
+- **Multiple-Choice Exams** involve choosing the correct answer from a set of options, essentially being a compilation of [quizzes](#quizzes).
```
Comment (Alex-deVis) [inline description.md:552]:
```suggestion
- **Multiple-Choice Exams** involve choosing the correct answer from a set of options, essentially being a compilation of ["Quizzes"](#quizzes).
```
We talked about this on discord, but I think it ruins the flow of reading.

**Review by Alex-deVis — APPROVED** [review]
We did it just in time.

File: description.md (hunk 1 above)
Comment (razvand) [reply description.md]: @Alex-deVis , see the new content. I preferred to use "Quizzes" only when wording it like `the "Quizzes" section`.

File: description.md [hunk 2]
```diff
@@ -327,7 +327,7 @@ The script will infer the fact that the input format is Markdown and the output
 python3 question_converter.py convert -i input_file.md -o output_file.mxml
 ```
 
-The converter may also be run by the builder if the educator desires to integrate questions within their published course and not use a third party hosting website, such as those outlined [in the Deployment section](#deployment).
+The converter may also be run by the builder if the educator desires to integrate questions within their published course and not use a third party hosting site, such as those outlined [in the Deployment section](#deployment).
```
Comment (Alex-deVis) [inline description.md:330]:
```suggestion
The converter may also be run by the builder if the educator desires to integrate questions within their published course and not use a third-party hosting site, such as those outlined in the ["Deployment" section](#deployment).
```

File: description.md [hunk 3]
```diff
@@ -441,11 +441,11 @@ We reference them as presented in the ["Types of Content" section](#types-of-con
   The lecturer is the one who uses the practical support materials to do live demonstrations and highlight concrete use cases.
   This may not be possible for certain topics, either because they are highly theoretical or because it is difficult to have a live setup.
   Where a live demonstration is not possible, a video recording (as a visual support material) may be used.
-  They consist of [**Guides**](#guides) used for demos.
+  They consist of **demos** based on [guides](#guides) used for demos.
 
 - **Assessment materials** are used to evaluate the knowledge and skills of participants, and to provide feedback on spots they need to focus more on.
   Typically they are to be used live during lectures, with their results providing immediate feedback and triggering discussions.
-  They consist of [**Quizzes**](#quizzes).
+  They consist of **quizzes** created from [questions](#questions) presented as quizzes.
```
Comment (Alex-deVis) [inline description.md:448]:
```suggestion
  They consist of [questions](#questions) presented as quizzes.
```

**Review by Alex-deVis — APPROVED** [review]
I only checked your changes and everything seems ok.
For the next milestone, we should review the usage of "quizzes" and "questions" as some sentences seemed rigid to me.

**Comment by razvand** [comment]
@Alex-deVis , I fixed based on your comments. Note that `site` is the preferred term to be used instead of `website`, as presented by the Markdown linter.

**Comment by razvand** [comment]
>  For the next milestone, we should review the usage of "quizzes" and "questions" as some sentences seemed rigid to me.

Yes, we will do that.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,2 @@
 This repository contains all the descriptive materials of the Open Education Hub teaching/learning methodology.
```
Comment (Alex-deVis) [inline README.md:1]:
```suggestion
# Methodology

This repository contains all the descriptive materials of the Open Education Hub teaching/learning methodology.
```
Superlinter requires a top-level header ([MD#041](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041---first-line-in-a-file-should-be-a-top-level-heading)).

**Review by Alex-deVis — APPROVED** [review]
Fix the last linter error and let's merge this.

File: description.md [hunk 5]
```diff
@@ -327,7 +327,7 @@ The script will infer the fact that the input format is Markdown and the output
 python3 question_converter.py convert -i input_file.md -o output_file.mxml
 ```
 
-The converter may also be run by the builder if the educator desires to integrate questions within their published course and not use a third party hosting website, such as those outlined [in the Deployment section](#deployment).
+The converter may also be run by the builder if the educator desires to integrate questions within their published course and not use a third-party hosting site, such as those outlined [in the" Deployment" section](#deployment).
```
Comment (teodutu) [inline description.md:330]:
```suggestion
The converter may also be run by the builder if the educator desires to integrate questions within their published course and not use a third-party hosting site, such as those outlined [in the "Deployment" section](#deployment).
```

File: description.md [hunk 6]
```diff
@@ -205,23 +205,23 @@ Out of the existing teaching platforms, a few highlights are:
...
+  These quizzes are called games and can be created either from scratch, or by using `Kahoot!`'s existing question bank.
+  Creating `Kahoot!` quizzes from scratch is difficult to automate because the platform requires submitting questions individually into a web page.
 
 Each of these platforms supports a different syntax for uploading questions.
-To make our questions comply with educational platforms easily, we need to store them in a generic format that is easily representable and automatically convertible to other required formats.
+To make our questions comply with educational platforms easily, we need to store them in a generic format that is easily presentable and automatically convertible to other required formats.
 For this reason and for the sake of consistency, we are storing questions in the same Markdown format as all of our written material.
 
-We are developing a modular and easily extendible converter that ports questions from Markdown to other formats.
+We are developing a modular and easily extendable converter that ports questions from Markdown to other formats.
```
Comment (teodutu) [inline description.md:221]: TBF they're both correct.

**Review by teodutu — CHANGES_REQUESTED** [review]
I'm bad at spotting typos, so I hope it's alright

**Pull request merged**